File: BaseLib/src/main/java/com/hjc/baselib/loadsir/TimeoutCallback.java
package com.hjc.baselib.loadsir;

import com.hjc.baselib.R;
import com.kingja.loadsir.callback.Callback;

/**
 * @Author: HJC
 * @Date: 2020/5/15 11:13
 * @Description: loadSir 网络超时
 */
public class TimeoutCallback extends Callback {

    @Override
    protected int onCreateView() {
        return R.layout.default_layout_timeout;
    }

}

File: library-base/src/main/java/com/hjc/library_base/loadsir/TimeoutCallback.kt
package com.hjc.library_base.loadsir

import com.hjc.library_base.R
import com.kingja.loadsir.callback.Callback

/**
 * @Author: HJC
 * @Date: 2020/5/15 11:13
 * @Description: loadSir 网络超时
 */
class TimeoutCallback : Callback() {

    override fun onCreateView(): Int {
        return R.layout.base_layout_timeout
    }
}